src/migration/1721903257825-Vordr.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Vordr1721903257825 implements MigrationInterface {
+  name = 'Vordr1721903257825'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "flags" jsonb NOT NULL DEFAULT '{"trustScore": 1, "vordr": false}'`);
+    await queryRunner.query(`ALTER TABLE "comment" ADD "flags" jsonb NOT NULL DEFAULT '{"vordr": false}'`);
+
+    await queryRunner.query(`CREATE INDEX "IDX_user_flags_vordr" ON post USING HASH (((flags->'vordr')::boolean))`);
+    await queryRunner.query(`CREATE INDEX "IDX_comment_flags_vordr" ON post USING HASH (((flags->'vordr')::boolean))`);
+
+    await queryRunner.query(`CREATE OR REPLACE FUNCTION comment_flags_update_vordr()
```

Comment:
Please add tests for the trigger

Reply:
👍

Reply:
Added tests
https://github.com/dailydotdev/daily-api/blob/536415643eaa3b3dbea8e5995cecf3b4e8304177/__tests__/triggers/user.ts#L39-L125